src/test/java/uk/gov/companieshouse/web/accounts/controller/smallfull/DirectorsReportApprovalControllerTest.java

```diff
@@ -109,13 +110,16 @@ private void setup() {
     }
 
     @Test
-    @DisplayName("Get directors report approval view - success path")
-    void getRequestSuccess() throws Exception {
+    @DisplayName("Get directors report approval view - success path - multiple approvers")
+    void getRequestSuccessMultipleApprovers() throws Exception {
 
         when(directorsReportApprovalService.getDirectorsReportApproval(TRANSACTION_ID, COMPANY_ACCOUNTS_ID))
                 .thenReturn(directorsReportApproval);
 
-        when(directorService.getAllDirectors(TRANSACTION_ID, COMPANY_ACCOUNTS_ID)).thenReturn(new Director[0]);
+        Director director = new Director();
+        director.setName(DIRECTOR_NAME);
```

Comment:
Should we add more than 1 director since it is a test for multiple approvers ?

Reply:
The secretary acts as the second available approver 👍